data/json/starting_missions.json

```
@@ -721,7 +721,8 @@
   {
     "id": "MISSION_LAST_DELIVERY",
     "type": "mission_definition",
-    "name": { "str": "Reach The Mansion And Finish This Delivery" },
+    "name": { "str": "Deliver Your Cargo" },
+    "description": "You’re not sure who orders takeout when the world’s ending, but that doesn’t matter.  Evacuation doesn’t matter.  Nothing, indeed, matters, save for safely delivering your precious cargo to its destination: a remote mansion.  May Food Person watch over you, soldier.",
```

```suggestion
    "description": "You’re not sure who orders takeout when the world’s ending, but that doesn’t matter.  Evacuation doesn’t matter.  Nothing, indeed, matters, save for safely delivering your precious cargo to its destination: a remote mansion.  May Foodperson watch over you, soldier.",
```
Foodperson is spelled without space